src/promptflow-devkit/promptflow/_sdk/_tracing.py

```diff
@@ -68,10 +59,22 @@ def _is_azure_ext_installed() -> bool:
         return False
 
 
+def _get_collection_id(collection: str) -> str:
```

Comment:
maybe include azure in the function name

Reply:
sure, will do this (as conflict is already there lol.

Reply:
updated